css-responsive/styles/sass/styles.scss

```diff
@@ -0,0 +1,149 @@
+@import "variables";
+@import "mixins";
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+html {
+  font-size: 62.5%;
+}
+
+body {
+  font-family: 'Poppins';
+  background-color: #FFFFFF;
+  font-size: 1.6rem;
+}
+
+.prueba {
+  width: 80vw;
+}
```

Comment:
This class was a custom class we created because the overflowX wasn't working for some reason, do you remember?
Attach an image of this bellow:
<img width="353" alt="Screen Shot 2023-02-28 at 16 44 03" src="https://user-images.githubusercontent.com/92554753/221961870-cb53f242-3561-4fcc-bb13-4559ced2b4da.png">

Shall we keep the clase and change the name?